config/sync/field.field.node.stanford_news.su_news_tags.yml

```diff
@@ -0,0 +1,29 @@
+uuid: cf2ecdeb-422a-4201-8a45-c6dbf5bff39c
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.su_news_tags
+    - node.type.stanford_news
+    - taxonomy.vocabulary.su_news_tag_filters
+id: node.stanford_news.su_news_tags
+field_name: su_news_tags
+entity_type: node
+bundle: stanford_news
+label: "News Tags/Filters\t"
+description: 'Enter the categories associated with this news article. These will be used in two ways: They will be displayed in the sidebar on the individual news page. They will also be used as filters for the Filtered Lists Paragraph view.'
```

Comment:
@mdyoung3 Let's change this help text to:
"These will be used as filters for the future Filtered Lists Paragraph for News. They do not display on the News item or News Teaser."